mixer/test/client/istio_authn_peer_jwt_bound_peer/authn_report_test.go

```diff
@@ -136,7 +158,7 @@ var reportAttributesOkGet = `
      "x-request-id": "*"
   },
   "request.size": 0,
-  "request.total_size": 817,
```

Comment:
Should we see some request.auth.* attributes?

Reply:
Yes, they are on lines 163~169.